### PR TITLE
Adds nightly e2e for covid cases, deaths, and hospitalizations

### LINKED ISSUE #2556: Nightly e2e for  covid cases
Nightly e2e for  covid cases

### LINKED ISSUE #2557: Nightly e2e for  covid deaths
Nightly e2e for  covid deaths

### LINKED ISSUE #2558: Nightly e2e for  covid hospitalizations
Nightly e2e for  covid hospitalizations

### PR DESCRIPTION
**Pull request by JDemlow**
# Description and Motivation
Closes #2558
Closes #2557
Closes #2556

## Has this been tested? How?

Tests pass locally

Code_90yXsFmzdS

## Types of changes

(leave all that apply)

- New content or feature

### CODE REVIEW

File: frontend/playwright-tests/covid_cases_deaths_hospitalizations.nightly.spec.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    .getByRole('heading', { name: 'Population vs. distribution' })
+    .click()
+  await page
+    .getByRole('heading', { name: 'Breakdown summary for COVID-' })
+    .click()
+  await page.getByText('Share this report:').click()
+  await page.getByText('COVID-19 cases', { exact: true }).click()
+  await page.getByText('Do you have information that').click()
+  await page.goto(
+    'http://localhost:3000/exploredata?mls=1.covid-3.00&group1=All'
```
Comment (benhammondmusic) [inline frontend/playwright-tests/covid_cases_deaths_hospitalizations.nightly.spec.ts:44]: We have to make sure that the base url isn't hard coded in; just use relative urls instead throughout

File: frontend/playwright-tests/covid_cases_deaths_hospitalizations.nightly.spec.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { test } from '@playwright/test'
+
+test('Covid Cases, Deaths, Hospitalizations', async ({ page }) => {
```
Comment (benhammondmusic) [inline frontend/playwright-tests/covid_cases_deaths_hospitalizations.nightly.spec.ts:3]: Let's actually split it into 3 test() functions inside this one single file. Then it'll be easier to identify a test failure in a particular datatype. Just have each new test start right before where you're having it go to local host (and make those relative urls)

**Review by benhammondmusic — CHANGES_REQUESTED** [review]

**Review by JDemlow — COMMENTED** [review]
The latest commit splits the test into three test blocks and fixes the relative URL issue

**Comment by JDemlow** [comment]
Ok! The following has been fixed:

- Test is split into three test blocks
- Fixed relative URL problem
- All refactored code has been reinserted
- Screenshot updated
- All checks pass

I believe we're good now 👍

**Review by benhammondmusic — APPROVED** [review]
Nice!!!

**Pull request merged**